### DISCUSSION TITLE
Regarding custom domain in Kutt

### DISCUSSION THREAD

**Discussion by Biswadeepgiri** [thedevs-network/kutt#757] [General]
The default behaviour of kutt is it’ll generate a link with the domain it’s hosted on But it has provision for creating domains like bitly and other platform has .So it must allow creating links with other domains.

How can I do that? @poeti8

**Comment by poeti8** [comment]
Add your domain from the settings page and then select it from the available domains in the advanced section when you're creating a link.

**Reply by jacobgraf** [reply]
Can I set up the admin interface at `kutt.mydomain.com` while using a different default short URL domain, like `shortdomain.com`? I set `DEFAULT_DOMAIN` to `shortdomain.com` in the .env file, but accessing `shortdomain.com` takes me to the admin interface, which isn't my goal. I want `shortdomain.com` to handle only short redirects and forward any non-existent short URLs to another site, while using `kutt.mydomain.com` solely for managing links.

Is this setup possible?

**Reply by chrislawso** [reply]
That should be possible by appropriately configuring your reverse proxy to perform the rules you require.
